docs/attachments/attachments-schema.yaml

```diff
@@ -15,28 +17,32 @@ definitions:
         size:
           type: number
         category:
-          type: string
-          ref: '#/definitions/resourcesCategory'
+          type: object
+          properties:
+            _id: 
+              type: string
+            name:
+              type: string
         createdAt:
           type: string
           format: date-time
         updatedAt:
           type: string
           format: date-time
+        resourceType:
+          type: string
   attachmentBody:
     type: object
     properties:
-      fileName:
+      id:
```

Comment:
You cannot change `id` field inside `patch` query. It is a bug on frontend that we send this field to the backend. But on backend we take only `fileName`, `description` and `category`
![image](https://github.com/user-attachments/assets/1fdd9774-bfd3-4b2b-a9be-cc48704bc7db)